product_barcode_constraint_per_company/__manifest__.py

```diff
@@ -1,18 +1,18 @@
-# Copyright (C) 2020-Today: GRAP (http://www.grap.coop)
-# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# Copyright 2023 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     "name": "Product Barcode Constraint per Company",
     "version": "14.0.1.0.1",
     "category": "Product",
-    "summary": "Change the product barcode constraint"
-    ", allowing the same barcode for differents companies",
-    "author": "GRAP, Odoo Community Association (OCA)",
+    "summary": """Change the product barcode constraint,
+                  allowing the same barcode for differents companies""",
+    "author": "GRAP, Cetmix, Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/stock-logistics-barcode",
     "license": "AGPL-3",
     "depends": [
-        "product",
+        "product_variant_company",
+        "product_multi_barcode",
```

Comment:
This new dependency is more problematic.

Reply:
ok, I'll fix it)